MODULE.bazel

```diff
@@ -1,39 +1,86 @@
 module(
```

Comment:
At this point we probably also need a release MODULE.bazel that gets bundled up into the final release archive. Otherwise these development dependencies are going to leak into the release Bzlmod definitions.

Reply:
Seems likely -- the `dev_dependency` attribute doesn't do enough work, especially if you want to use the repo for development work.